strapi/strapi
# feat(content-releases): add create release action to cm edit view

### What does it do?

- Add the injection zone for the content manager edit view
- Add the create release action button and form
- Add Release Action type radio button component
- Refactor:
   - Share the release schema between server and AddReleaseDialog (open to discussion 🙃 )
   - Use yup.strapID in Release Action schema and update validation to handle strings

### Why is it needed?

To add an entry to a release from the CM edit view

### How to test it?

- Go to the content manager with permissions to create a release action
- You should see the injections zone for releases with a button to add to release
- Click the button and a modal should open
- Select a release and the action to take
- Hit continue to send the request
